microcosm_pubsub/conventions.py

```diff
@@ -0,0 +1,98 @@
+"""
+Convention-driven pubsub.
+
+"""
+from enum import Enum, unique
+
+from inflection import underscore
+from marshmallow import fields
+
+from microcosm_pubsub.codecs import PubSubMessageSchema
+
+
+@unique
+class LifecycleChange(Enum):
+    """
+    CRUD lifecycle changes that may be announced via pubsub.
+
+    Note that `Changed` is strongly discouraged; use immutable resources instead.
+
+    """
+    Changed = u"changed"
+    Created = u"created"
```

Comment:
I went with `Created` instead of `New`; this means that these messages will not interchange with the existing message schemas and we'll have to change both sides of every interaction.

I was torn about this; the migration to not defining schemas in every repo will be easier if we have backwards compatibility. However, these adjectives are a lot more consistent/clear with the rest of the system.